### DISCUSSION TITLE
solweig outputs data from 1-23 points. The temperature at 6 points is too high

### DISCUSSION THREAD

**Discussion by hhh321747** [UMEP-dev/UMEP#651] [Q&A]
solweig outputs data from 1 to 23 points. The temperature at 6 points is too high, and the data at other times is normal
![c2e7f51a3244a1377612dfc85565624](https://github.com/user-attachments/assets/672a3613-3d0d-456d-87ae-c2b25b41fb00)
![c49d3f61fe2430ffd30ff80327c491c](https://github.com/user-attachments/assets/257bfa25-9738-4c16-b2a4-0e054c439af3)

**Comment by biglimp** [comment]
You have -999 in your Kdn column. This is required input data for solweig.
